BookingService/BookingService.Application/DTOs/LoginDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace BookingService.Application.DTOs
+{
+    public class LoginDto

```

Comment:
I suggest using `record` for your DTO because they fit dto pattern perfectly: data classes that are immutable by design